source/app/service-providers/fsal/util/file-parser.ts

```diff
@@ -71,7 +71,7 @@ export default function getMarkdownFileParser (
     const ast = md2ast(content)
 
     const tags = extractASTNodes(ast, 'ZettelkastenTag') as ZettelkastenTag[]
-    file.tags = tags.map(tag => tag.value)
```

Comment:
Before doing this, we would need to discuss if the `#` sign is important for the tags in the AST. If it is not, we should remove it in the AST parser already, not here.

Look here: https://github.com/Zettlr/Zettlr/blob/e920dee0741641c82707e04a30d9002ecb74d00a/source/common/modules/markdown-utils/markdown-ast.ts#L877

Reply:
Just thought about it: We've always removed the `#`, so you can add the `substring(1)` (or `slice(1)`, which I've been using lately) to the AST parser itself which I linked.